vueui/src/assets/theme-nano.scss

```diff
@@ -130,22 +162,32 @@
     }
     
     .uiTitleWrapper {
-        background: url(resources/theme-nano/uiTitleBackground.png) repeat center;
+        background-color: $header-bg;
         box-shadow: 0px 0px 8px 4px rgba(0,0,0,0.5);
         .uiTitleText {
             color: #E9C183;
         }
     }
 
-    .uiTitleFluff {
-        display: block;
-        position: relative;
-        float: right;
-        top: 4px;
-        right: 10px;
-        width: 42px;
-        height: 24px;
-        background: url(resources/theme-nano/uiTitleFluff.png) 50% 50% no-repeat;
+    .uiTitleClose {
+        position: absolute;
+        top: -1px;
+        right: 0;
+        width: 45px;
+        height: 32px;
+        font-size: 20px;
+        line-height: 31px;
+        text-align: center;
+
+        color: color.change($text-color, $alpha: 0.5);
+        background-color: $header-bg;
```

Comment:
The close element is on top of the title wrapper, correct? In that case it shouldn't need background-color set, just keep it transparent and we can even have images in there.

Additionally, clickable elements like this should probably have `cursor: pointer` set.